### ISSUE TITLE
Generating Feature Embeddings using ArcFace+ResNet100

### ISSUE THREAD

**Issue by sohaibjawad** [YirongMao/NAN#6]
Hi! I am following your repo as is, but the only difference is that I am computing feature embeddings using ArcFace+Resnet100 pre-trained model. Could you please guide me through the steps you took to generate .mat files of feature embeddings? I am using the face bounding box information given in train split files and then computing it's embedding. The TAR@FAR values I am getting are significantly lower than yours. Waiting for your response! 

Thanks

**Comment by YirongMao** [comment]
My face model is ResNet-34, trained on WebFace with softmax loss. If your face model is robust enough, NAN may be unable to improve the performance.

**Comment by YirongMao** [comment]
> Hi! I am following your repo as is, but the only difference is that I am computing feature embeddings using ArcFace+Resnet100 pre-trained model. Could you please guide me through the steps you took to generate .mat files of feature embeddings? I am using the face bounding box information given in train split files and then computing it's embedding. The TAR@FAR values I am getting are significantly lower than yours. Waiting for your response!
> 
> Thanks

My face model is ResNet-34, trained on WebFace with softmax loss. If your face model is robust enough, NAN may be unable to improve the performance.

**Comment by sohaibjawad** [comment]
I see. And did you preprocess the images like image alignment etc? If yes
which technique did you use?

**Comment by YirongMao** [comment]
> I see. And did you preprocess the images like image alignment etc? If yes which technique did you use?
> […](#)

I only detect face with MTCNN(https://github.com/ipazc/mtcnn). No other preprocess steps.

**Comment by sohaibjawad** [comment]
Okay thanks I'll give it a try

**Comment by sohaibjawad** [comment]
Could you please send me the Resnet-34 model that you used? I want to
validate my python script where I am generating feature embeddings and
splitting the dataset into templates

**Comment by sohaibjawad** [comment]
How did you compute feature embeddings using mtcnn for subjects where there are more than 1 person in the frame?

**Comment by tuantran23012000** [comment]
> Hi! I am following your repo as is, but the only difference is that I am computing feature embeddings using ArcFace+Resnet100 pre-trained model. Could you please guide me through the steps you took to generate .mat files of feature embeddings? I am using the face bounding box information given in train split files and then computing it's embedding. The TAR@FAR values I am getting are significantly lower than yours. Waiting for your response!
> 
> Thanks

Can you provide IJBA dataset?

**Comment by hairenaa** [comment]
我运行了您的代码但accuracy显示很小，甚至是0，这是为什么呢？，另外我并不能清晰地看到原始数据，请问您有IJB-A的数据集吗？
I ran your code but the accuracy shows very little, even 0, why is that? , In addition, I can't see the raw data clearly, do you have an IJB-A dataset
